Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -17,7 +17,7 @@
 
 ''' ADVANCED GLOBAL PARAMETERS '''
 
-FAILURE_SLEEP = 15  # sleep between consecutive failures events fetch
+FAILURE_SLEEP = 20  # sleep between consecutive failures events fetch
```

Comment:
Shouldn't this be 30 seconds, to round up if needed?

Reply:
20 seconds should be enough (the third request should succeed in all cases)